frontend/model/contracts/group.js

```diff
@@ -303,6 +307,43 @@ sbp('chelonia/defineContract', {
     },
     groupDistributionEvents (state, getters) {
       return getters.currentGroupState.distributionEvents
+    },
+    getChatRooms (state, getters) {
+      const active = {}
+      const deleted = {}
+      for (const cID in getters.currentGroupState.chatRooms) {
+        if (!getters.currentGroupState.chatRooms[cID].deletedDate) {
+          active[cID] = getters.currentGroupState.chatRooms[cID]
+        } else {
+          deleted[cID] = getters.currentGroupState.chatRooms[cID]
+        }
+      }
+      return { active, deleted, all: { ...active, ...deleted } }
+    },
+    getChatRoomIDsInSort (state, getters, rootState, rootGetters) {
+      const chatRooms = getters.getChatRooms.active
+      return Object.keys(chatRooms)
+        .map(chatRoomID => ({
+          name: chatRooms[chatRoomID].name,
+          privacyLevel: chatRooms[chatRoomID].privacyLevel,
+          joined: rootGetters.isJoinedChatRoom(chatRoomID),
+          id: chatRoomID
+        })).filter(details => details.privacyLevel !== CHATROOM_PRIVACY_LEVEL.PRIVATE || details.joined).sort((former, latter) => {
+          const formerName = String(former.name).toLowerCase()
+          const latterName = String(latter.name).toLowerCase()
```

Comment:
What is the reason to wrap `former.name` and `latter.name` in `String`? Aren't they already strings?

Reply:
Yes. Good catch. I think it is unnecessary.